.github/ISSUE_TEMPLATE/suggestion.yml

```diff
@@ -0,0 +1,26 @@
+name: Suggest a feature
+description: Suggest a new feature, behavior or improvement to scikit-image.
+title: "<Please write a comprehensive title summarizing your request>"
+labels: [":pray: Feature request"]
```

Comment:
I would like to rename this label to ":pray: Suggestion". I think it sounds a little less demanding and friendlier than "request". Stripping the feature makes it more generally applicable, e.g. to improvements, new ideas or behavior, ...